src/realm/query_expression.hpp

```diff
@@ -2598,6 +2768,11 @@ class Columns : public Subexpr2<T> {
         return m_sg ? get_column_base().get_column_index() : m_column_ndx;
     }
 
+    SizeOperator<Size<T>> size()
+    {
+        return SizeOperator<Size<T>>(this->clone(nullptr));
```

Comment:
I am just trying to be as general as possible. If you think this will never be used, I can certainly remove it.